boot/zephyr/include/mcuboot_config.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
I know this is an RFC, but can you please use include guards for the real submission?